integration_tests/conftest.py

```diff
@@ -12,6 +12,9 @@ def sim() -> InteractiveContext:
     """Initialize a simulation for use in tests"""
     simulation = InteractiveContext(paths.MODEL_SPEC_DIR / "model_spec.yaml", setup=False)
     simulation.configuration.population.population_size = 20_000
+    simulation.configuration.input_data.artifact_path = str(
```

Comment:
The entire test suite is passing with this change...so is it time to go ahead and start using this artifact as the "official" one? (keeping in mind it won't actually get to `main` for a bit while we work on the epic

Reply:
yes, that sounds good